### PR TITLE
[iOS] RefreshControl doesn't render when initial refreshing state is true

### PR DESCRIPTION
**Pull request by doochik**
Example (index.ios.js):

```
'use strict';

import React from 'react';
import {ListView, RefreshControl, Text, View} from 'react-native';

class BugExample extends React.Component {

    render() {
        return (
            <ListView
                style={{backgroundColor: 'red'}}
                contentContainerStyle={{backgroundColor: 'green'}}
                refreshControl={
                    <RefreshControl
                        refreshing={true}
                        onRefresh={() => {}}
                    />
                }
                dataSource={
                    new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2}).cloneWithRows(['a', 'b', 'c'])
                }
                renderRow={(item) => {
                    return (
                        <View>
                            <Text>{item}</Text>
                        </View>
                    );
                }}
            />
        );
    }
}
```

RN version: 0.34, 0.35-rc
iOS version (emulator): 9.3, 10

If you run this example you'll see that ScrollView has offest for RefreshControl, but RefreshControl doesn't visible. Any actions (eg, scroll) make RefreshControl visible.

This happens because first render is blocked by this flag https://github.com/facebook/react-native/blob/016f47c7f5c1009a0340360ec0af92a14cae462f/React/Views/RCTRefreshControl.m#L56.

I've checked my fix in RN 0.34, 0.35-rc and all works fine after patch.

### CODE REVIEW

File: React/Views/RCTRefreshControl.m [hunk 1]
```diff
@@ -119,9 +113,7 @@ - (void)setRefreshing:(BOOL)refreshing
     if (refreshing) {
       // If it is the initial render, beginRefreshing will get called
       // in layoutSubviews.
```
Comment (GantMan) [inline React/Views/RCTRefreshControl.m]: These comments don't make sense when you short-circuit the IF condition.  They should be removed/modified.

Comment (doochik) [inline React/Views/RCTRefreshControl.m]: Yes, this comment is obvious, remove it.

**Comment by javache** [comment]
cc @mmmulani

**Comment by shixukai** [comment]
exactly the same issue with me

**Comment by doochik** [comment]
@javache @mmmulani Hi guys! Any chance to see this PR merged?

**Comment by doochik** [comment]
@javache @mmmulani Can you review my PR? I have to patch source every time I update RN :(

**Comment by mmmulani** [comment]
lgtm

**Pull request closed without merging**